parachains/chain-specs/collectives-polkadot-genesis.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "Collectives",
+  "id": "collectives_polkadot",
+  "chainType": "Live",
+  "bootNodes": [
+    "/dns/polkadot-collectives-collator-ew6-0.polkadot.io/tcp/30334/p2p/12D3KooWKgrivNBkMswNNNYqUkhEBvnyRzt6TLPQ2shu4eoPuuRi",
+    "/dns/polkadot-collectives-collator-ew6-1.polkadot.io/tcp/30334/p2p/12D3KooWQen2oX7hWqhUiJ4sxj7WA4Qa3WoAmz6rQbmYPCYjeCk9",
+    "/dns/polkadot-collectives-collator-uw1-0.polkadot.io/tcp/30334/p2p/12D3KooWJXwJS1d2UcdkKd8a6vM4TFB79gvtHtsp4bm4c8fYdpY7",
+    "/dns/polkadot-collectives-collator-uw1-1.polkadot.io/tcp/30334/p2p/12D3KooWRjdypZdMT9rjajYM2L2P5mYaxPGErP9ExGBsY3PBdYDy",
+    "/dns/polkadot-collectives-connect-ew6-0.polkadot.io/tcp/30334/p2p/12D3KooWLDZT5gAjMtC8fojiCwiz17SC61oeX2C7GWBCqqf9TwVD",
+    "/dns/polkadot-collectives-connect-ew6-1.polkadot.io/tcp/30334/p2p/12D3KooWC9BwKMDyRUTXsE7teSmoKMgbyxqAp3zi2MTGRJR5nhCL",
+    "/dns/polkadot-collectives-connect-uw1-0.polkadot.io/tcp/30334/p2p/12D3KooWPrJ9VTn3GEs2e7GQs4zoEFiTFcjXFNbQ2iDxFDQAbstQ",
+    "/dns/polkadot-collectives-connect-uw1-1.polkadot.io/tcp/30334/p2p/12D3KooWGFYW6hJYn3pkpJawyMk6souXh7sznK8yvPmVQ7ktfHbV",
+    "/dns/polkadot-collectives-connect-ew6-0.polkadot.io/tcp/443/wss/p2p/12D3KooWLDZT5gAjMtC8fojiCwiz17SC61oeX2C7GWBCqqf9TwVD",
+    "/dns/polkadot-collectives-connect-ew6-1.polkadot.io/tcp/443/wss/p2p/12D3KooWC9BwKMDyRUTXsE7teSmoKMgbyxqAp3zi2MTGRJR5nhCL",
+    "/dns/polkadot-collectives-connect-uw1-0.polkadot.io/tcp/443/wss/p2p/12D3KooWPrJ9VTn3GEs2e7GQs4zoEFiTFcjXFNbQ2iDxFDQAbstQ",
+    "/dns/polkadot-collectives-connect-uw1-1.polkadot.io/tcp/443/wss/p2p/12D3KooWGFYW6hJYn3pkpJawyMk6souXh7sznK8yvPmVQ7ktfHbV"
+  ],
+  "telemetryEndpoints": null,
+  "protocolId": null,
+  "properties": {
+    "ss58Format": 0,
+    "tokenDecimals": 10,
+    "tokenSymbol": "DOT"
+  },
+  "relay_chain": "polkadot",
+  "para_id": 1001,
```

Comment:
We're happy about the para_id? (Looks like encointer is 1003.)

Reply:
Yes, Encointer is not on Polkadot.

Reply:
Oh yeah good point.